frontstage/views/surveys/help/surveys_help.py

```diff
@@ -393,6 +290,58 @@ def send_help_message(session, survey_ref, ru_ref):
         return redirect(url_for("surveys_bp.get_survey_list", tag="todo"))
 
 
+def abort_help_if_session_not_set():
+    """
+    Aborts the process if flask session attributes are not set
+    """
+    if flask_session["help_survey_ref"] is None or flask_session["help_ru_ref"] is None:
```

Comment:
I don't think this will work as you expect.  If you  try and get a key using the `dict[key]` notation then it'll raise a `KeyError` if it's not found.

These both should be  changed to:
```python
if flask_session.get("help_survey_ref") is None or flask_session.get("help_ru_ref") is None:
```
as `.get(key)` will return None if it doesn't find it.

Reply:
Yeah, just checked it (by going to `http://localhost:8082/surveys/help/help-completing-this-survey` directly) and you get the `An error has occurred` page (which I think you get on a 500 error) rather then the 'Not found' you were probably expecting